src/pages/ExperienceBeginning.svelte

```diff
@@ -1 +1,14 @@
-<h2>Start the experience</h2>
+<script>
+  import { _ } from 'svelte-i18n'
+  import NextButton from '../components/NextButton.svelte'
+</script>
+
+<section>
```

Comment:
Maybe would it be better to have `<section><slot></slot></section>` in the `Page` component or you will have to include it on each page.

Reply:
Will be done in the next PR :)